ImperatorToCk2/src/main/java/ImperatorToCK2/Characters.java

```diff
@@ -1,210 +1,210 @@
-package ImperatorToCK2; 

```

Comment:
Leave the package definition in.

Reply:
That might be why it's not building, lol, I think that my IDE removed them. I'll add them back tomorrow